config/database.yml

```
@@ -0,0 +1,37 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= (ENV['PG_POOL_SIZE'] || ENV.fetch('RAILS_MAX_THREADS', 8)).to_i %>
+  url: <%= ENV.fetch('DATABASE_URL', 'postgresql://talk:talk@localhost') %>
+
+development:
+  <<: *default
+  database: talk_development
+
+panoptes_development:
+  <<: *default
+  database: talk_panoptes_development
+  url: <%= ENV.fetch('PANOPTES_DATABASE_URL', 'postgresql://talk:talk@localhost') %>
```

not blocking - these repeating `panoptes_*` envs could have been added to their own default e.g. 
```
panoptes_default: &panoptes_default
  <<: *default
    url: <%= ENV.fetch('PANOPTES_DATABASE_URL', 'postgresql://talk:talk@localhost') %>
```
```suggestion
  <<: *panoptes_default
  database: talk_panoptes_development
```

---

✔️ , why not!